packages/core/src/web/app/components/beambox/LeftPanel/index.module.scss

```diff
@@ -31,6 +31,14 @@ $icon-gap: 13px;
   }
 }
 
+.ai-generate {
+  font-size: 32px;
+
+  :hover {
+    color: #000000;
+  }
+}
+
```

Comment:
`hover color` seems unnecessary because `LeftPanelButton` also have hover color.

Also I feel we can move `font-size: 32px` in `ai-generate` and `beamy` to `LeftPanelButton.module.scss`. The existing 22px seems not used